app/views/catalog/_index_map.html.erb

```diff
@@ -2,6 +2,7 @@
                        {data:{viewpoint: set_viewpoint(geojson_features),
                               searchcontrol: true,
                               catalogpath: catalog_index_path,
-                              placenamefield: blacklight_config.view.maps.placename_field
+                              placenamefield: blacklight_config.view.maps.placename_field,
+                              clustercount:'hits'
```

Comment:
Probably a good idea. I will add something in the documentation about the properties that can be passed in the data hash.